Quorum/ipfs_validator.py

```diff
@@ -0,0 +1,69 @@
+from Quorum.utils.chain_enum import Chain
+import Quorum.config as config
+from Quorum.apis.block_explorers.chains_api import ChainAPI
+from Quorum.apis.llms.claude_conversation import ClaudeConversation
+
+import requests
+import argparse
+import re
+from pathlib import Path
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description='Compare ipfs with actual payload.')
+    parser.add_argument('--proposal_id', type=int, help='The id of the proposal.')
+    parser.add_argument('--chain', type=str, choices=[chain.value for chain in Chain], help='Blockchain chain.')
+    parser.add_argument('--proposal_address', type=str, help='Ethereum proposal address.')
```

Comment:
Replace str type with actual validation for the propsal_address.

Reply:
Mmm, how would you "actually validate" the address? I took this arg exactly from `check_proposal`...

Reply:
regex for address should be enough